lib/internal/test_runner/utils.js

```diff
@@ -74,10 +77,21 @@ function convertStringToRegExp(str, name) {
   }
 }
 
+let _module;
+function getTestReporter(reporter) {
+  if (getOptionValue('--test-reporter')) {
+    _module ??= new Module('node:test');
+    const reporters = _module.require(getOptionValue('--test-reporter'));
```

Comment:
yeah, we want to support built-in reporters `require` does not work for built-in modules:
` Missing internal module 'node:test/reporter/spec'`

Reply:
Since we will only have one or two built in reporters, maybe we should load them separately and use the normal require/import for userland modules instead of dragging chunks of the module loader into the test runner.

Reply:
the "normal"  `require` dosent seem to let me import either internal or "local" reporters, it errors with `Missing internal module`